core/src/repair/quic_endpoint.rs

```diff
@@ -168,28 +174,35 @@ pub(crate) fn close_quic_endpoint(endpoint: &Endpoint) {
     );
 }
 
-fn new_server_config(cert: Certificate, key: PrivateKey) -> Result<ServerConfig, rustls::Error> {
+fn new_server_config(
+    cert: CertificateDer<'static>,
+    key: PrivateKeyDer<'static>,
+) -> Result<ServerConfig, rustls::Error> {
     let mut config = rustls::ServerConfig::builder()
-        .with_safe_defaults()
         .with_client_cert_verifier(Arc::new(SkipClientVerification {}))
         .with_single_cert(vec![cert], key)?;
     config.alpn_protocols = vec![ALPN_REPAIR_PROTOCOL_ID.to_vec()];
-    let mut config = ServerConfig::with_crypto(Arc::new(config));
+    let quic_server_config = QuicServerConfig::try_from(config)
+        .map_err(|_err| rustls::Error::InvalidCertificate(CertificateError::BadSignature))?;
+
+    let mut config = ServerConfig::with_crypto(Arc::new(quic_server_config));
     config
         .transport_config(Arc::new(new_transport_config()))
-        .use_retry(true)
```

Comment:
Out of curiosity, why disable retry? (Retries are cheap and don't require any X25519 or Ed25519 cryptography)

Reply:
use_retry is removed from the Config, it is now implemented by the application layer.